src/boards/include/boards/vgaboard.h

```diff
@@ -12,21 +12,8 @@
 #ifndef _BOARDS_VGABOARD_H
 #define _BOARDS_VGABOARD_H
 
-#ifndef PICO_DEFAULT_UART
-#define PICO_DEFAULT_UART 1
-#endif
-
-#ifndef PICO_DEFAULT_UART_TX_PIN
-#define PICO_DEFAULT_UART_TX_PIN 20
-#endif
-
-#ifndef PICO_DEFAULT_UART_RX_PIN
-#define PICO_DEFAULT_UART_RX_PIN 21
-#endif
-
-#ifndef PICO_DEFAULT_LED_PIN
-#define PICO_DEFAULT_LED_PIN 25 // same as Pico
-#endif
+// vgaboard has a Pico on it.
+#include "boards/pico.h"
```

Comment:
Nice :+1: 
...although I guess replacing `#define PICO_DEFAULT_UART_TX_PIN 20` with `#include "boards/pico.h"` means that any programs built for vgaboard now have a different `PICO_DEFAULT_UART_TX_PIN` ? (although perhaps this doesn't matter? :shrug: .... although looking at page 26 of https://datasheets.raspberrypi.org/rp2040/hardware-design-with-rp2040.pdf I see that it _does_ break out GPIO20 (as UART1_TX) to a UART header.)

Reply:
oops